README.md

```diff
@@ -208,7 +198,7 @@ We can check the configuration needed
 
 For PostgreSQL password, we use the .env file (remember to use the same password as the one you used when running the docker container)
 ``` 
-echo 'export TARGET_POSTGRES_PASSWORD=password' > .env
+echo 'export TARGET_POSTGRES_PASSWORD=londonpie' > .env
```

Comment:
I blindly copy pasted this line and failed because the Docker container was using another password